### PR TITLE
Skrell surgeons no longer spawn with a surgeon cap

### PR DESCRIPTION
**Pull request by carpevenenum**
Skrell headwear needs to be looked into mechanically overall, but the surgeon cap is particularly bad on them, so I removed it from spawning on them for now. Not that they need it, anyway.

### CODE REVIEW

File: html/changelogs/carpevenenum - skrellsurgeons.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (listerla) [inline html/changelogs/carpevenenum - skrellsurgeons.yml:30]: You should probably change this.

**Comment by carpevenenum** [comment]
!review

File: code/game/jobs/job/medical.dm [hunk 2]
```diff
@@ -114,7 +114,11 @@
 	uniform = /obj/item/clothing/under/rank/medical/blue
 	suit = /obj/item/clothing/suit/storage/toggle/labcoat/surgeon
 	shoes = /obj/item/clothing/shoes/surgeon
-	head = /obj/item/clothing/head/surgery/blue
+
+/datum/outfit/job/doctor/surgeon/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if (isskrell(H) == FALSE)
+		H.equip_to_slot_or_del(new /obj/item/clothing/head/surgery/blue(H), slot_head)
```
Comment (NonQueueingMatt) [inline code/game/jobs/job/medical.dm:121]: This isn't an acceptable solution I don't think. It's too hard-coded for my liking. I'd just remove the cap in general and place them in ORs.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/jobs/job/medical.dm (hunk 2 above)
Comment (carpevenenum) [reply code/game/jobs/job/medical.dm:121]: That's how tajara and unathi procs work as well. It's a bandaid solution until actual Skrell headgear mechanics are implemented.

Comment (NonQueueingMatt) [reply code/game/jobs/job/medical.dm:121]: I hate shitcode.

**Review by NonQueueingMatt — APPROVED** [review]

File: code/game/jobs/job/medical.dm [hunk 3]
```diff
@@ -114,7 +114,11 @@
 	uniform = /obj/item/clothing/under/rank/medical/blue
 	suit = /obj/item/clothing/suit/storage/toggle/labcoat/surgeon
 	shoes = /obj/item/clothing/shoes/surgeon
-	head = /obj/item/clothing/head/surgery/blue
+
+/datum/outfit/job/doctor/surgeon/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if (isskrell(H) == FALSE)
```
Comment (Geevies) [inline code/game/jobs/job/medical.dm:120]:
```suggestion
	if(!isskrell(H))
```

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**